src/gen/primitives/string.rs

```diff
@@ -34,8 +33,7 @@ impl Renderable for StringCodeType {
                 }
 
                 static int allocationSize([String value = ""]) {
-                    // FIXME: doing this twice for every string is bad
-                    return utf8.encoder.convert(value).length + 4; // Four additional bytes for the length data
+                    return utf8.encoder.convert(value).length + 4;
```

Comment:
I'd prefer we kept the code comment so people reading it understand what that number is about:
```suggestion
                    return utf8.encoder.convert(value).length + 4; // Four additional bytes for the length data
```

Reply:
yeah, I will bring it back